packages/react-components/src/AccessibilityScenarios/Button.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Button } from '@fluentui/react-button';
+import { Scenario, APP_TITLE, APP_TITLE_SEPARATOR } from './utils';
+
+export const CalendarButtonsAccessibilityScenario: React.FunctionComponent = () => {
+  const onSaveButtonClick = React.useCallback(() => {
```

Comment:
No need to wrap this into `useCallback` pass it directly into `onClick` instead or hold it in a simple `const`.

Reply:
Okay, removed the useCallback.